locales/en/fluid-queue.json

```diff
@@ -0,0 +1,100 @@
+  "modRemoveNoArgument": "You can use !remove <username> to kick out someone else's $t(level, {\"count\": 1}).",
+  "modRemoveNotFound": "No $t(level, {\"count\": 0}) from {{usernameArgument}} were found in the queue.",
+  "modRemoveRemoved": "{{usernameArgument}}'s $t(level, {\"count\": 1}) has been removed from the queue.",
+  "removeCurrentLevel": "Sorry, we're playing that level right now!",
+  "submitterNotFound": "{{submitter}}, looks like you're not in the queue. Try !add XXX-XXX-XXX.",
+  "removeRemoved": "{{submitter}}, your level has been removed from the queue.",
+  "replaceReplaced": "{{level.submitter}}, your level in the queue has been replaced with {{level}}.",
+  "eligibleUsers": "Eligible users: {{list}}",
+  "eligibleUsersTime": "Eligible users time: {{list}}",
+  "consolePrintWeight": "{{submitter}}'s weight is {{weight}} with totalWeight {{totalWeight}}",
+  "puntNoLevel": "The nothing you aren't playing cannot be punted.",
+  "puntPunted": "Ok, adding the current level back into the queue.",
+  "dismissNoLevel": "The nothing you aren't playing cannot be dismissed.",
+  "dismissDismissed": "Dismissed {{data.current_level}} submitted by {{data.current_level.submitter}}.",
+  "randomNumber": "Random number: {{randomNumber}}",
+  "currentCumulativeTime": "Current cumulative time: {{gettingThereSomeday}}",
+  "consoleIndexChosen": "Chosen index was {{levelIndex}} after a cumulative time of {{gettingThereSomeday}}",
+  "persistenceActivated": "Activated automatic queue persistence.",
+  "persistenceDeactivated": "Deactivated automatic queue persistence.",
+  "persistenceSaved": "Successfully persisted the queue state.",
+  "persistenceError": "Error while persisting queue state, see logs.",
+  "persistenceReloaded": "Reloaded queue state from disk.",
+  "persistenceInvalid": "Invalid arguments. The correct syntax is !persistence {on/off/save/load}.",
+  "queueEmpty": "There are no $t(level, {\"count\": 0}) in the queue.",
+  "queueListCurrent": "{{onlineLength}} online: {{- current.submitter}} (current), {{- onlineList, list}}... ({{levels.offline.length}} offline)",
+  "queueListCurrentOnly": "{{onlineLength}} online: {{- current.submitter}} (current)... ({{levels.offline.length}} offline)",
+  "queueListNoCurrent": "{{onlineLength}} online: (no current level), {{- onlineList, list}}... ({{levels.offline.length}} offline)",
+  "submitterPercentage": "{{submitter}} ({{percent, number}})",
+  "genericPercentage": "{{percent, number}}",
+  "weightedListCurrent": "{{onlineLength}} online: {{- current.submitter}} (current), {{- onlineList, list}}... ({{weightedList.offlineLength}} offline)",
+  "weightedListCurrentOnly": "{{onlineLength}} online: {{- current.submitter}} (current)... ({{weightedList.offlineLength}} offline)",
+  "weightedListNoCurrent": "{{onlineLength}} online: (no current level), {{- onlineList, list}}... ({{weightedList.offlineLength}} offline)",
```

Comment:
I see you have removed the `etc.`.
There are still those `...` which I know have been here always, but I think we do not need those `...`, because there is no semantic to it. (It doesn't even say that there are more levels in the queue than the ones displayed, because it there are 2 levels in the queue it will display both of those levels and then still those dots)

I think we should remove the dots and maybe there could be a way to tell that there are more levels following (levels.online.length > 5) which could be the dots or maybe an `etc.`.

Reply:
Ah, I forgot to mention the `etc`...

And it's probably good I did. I wrote out a bunch here as to why I got rid of it, but it got me to think about it more, and now there's still no `etc` but I've updated it to conditionally print an ellipsis through some (what I think is clever) nesting of localization strings. I've also added a test to test the ellipses, with some very unimaginative usernames and some random level codes picked from my Discord.